playbook/app/pb_kits/playbook/pb_table/styles/_hover.scss

```diff
@@ -9,17 +9,17 @@ $border_hover_color_dark: lighten($border_dark, 5%);
   &.table-lg {
     &:not(.no-hover) {
       border-collapse: collapse;
-      tbody {
-        tr {
+      tbody, .tbody {
+        tr, .tr {
           box-shadow: 0 0 0 $white;
           transition: box-shadow $transition-speed ease;
-          td {
+          td, .td {
             border-top-color: transparent;
             border-top-width: 0;
             transition: all $transition-speed ease;
           }
           @media (hover:hover) {
-            td {
+            td, .td {
```

Comment:
These classes are too weak and can encourage style bleed. I suggest making them stronger. Even `pb_td` is better. These are internal classes so making them verbose is ok here. Following SMACCS I would prefix them with the parent class:

`.pb_table`
`. pb_table_thead`
`. pb_table_td`
`. pb_table_tr` or `. pb_table_row`

ect.